static/js/HitchhikerJS.js

```diff
@@ -34,9 +34,16 @@ export { ManualInitializer };
 import RuntimeConfigReceiver from './runtime-config-receiver';
 export { RuntimeConfigReceiver };
 
+import DeferredPromise from './deferred-promise';
+window.AnswersInitialized = new DeferredPromise((resolve, reject) => {
```

Comment:
What do you think of putting this AnswersInitialized promise inside of `AnswersExperience` rather than on the window?

Reply:
Do you think it would be clearer if we renamed `AnswersInitialized` to `AnswersInitializedPromise`?